unity-renderer/Assets/DCLServices/CameraReelService/ScreenshotMetadata.cs

```diff
@@ -0,0 +1,114 @@
+﻿using DCL;
+using DCLServices.WearablesCatalogService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace UI.InWorldCamera.Scripts
+{
+    [Serializable]
+    public class ScreenshotMetadata
+    {
+        public string userName;
+        public string userAddress;
+        public string dateTime;
+        public string realm;
+        public Scene scene;
+        public VisiblePlayers[] visiblePeople;
+
+        public static ScreenshotMetadata Create(DataStore_Player player, IAvatarsLODController avatarsLODController, Camera screenshotCamera)
+        {
+            Player ownPlayer = player.ownPlayer.Get();
+            Vector2Int playerPosition = player.playerGridPosition.Get();
+
+            var metadata = new ScreenshotMetadata
+            {
+                userName = UserProfileController.userProfilesCatalog.Get(ownPlayer.id).userName,
+                userAddress = ownPlayer.id,
+                dateTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds().ToString(),
+                realm = DataStore.i.realm.realmName.Get(),
```

Comment:
does this work fine if we are connected to a world?

Reply:
as for now we don't use this data and taking screenshot in world seems to be fine